src/docs/arch-hs.dox

```diff
@@ -1,8 +1,123 @@
 /*! @arch_page arch-hs History Store
 
-The History Store in WiredTiger tracks old (all but the latest committed)
-versions of records.  By having these records in storage separate from
+The history store in WiredTiger tracks old (all but the latest committed)
```

Comment:
The phrase (all but the latest committed) is misleading, by that definition we'd have an ever increasing history store. It should be updated to say something like (all that are required to service older readers).

Reply:
Fixed.